client/gutenberg/extensions/map-block/add-point/index.js

```diff
@@ -0,0 +1,73 @@
+/**
+ * Wordpress dependencies
+ */
+
+import { __ } from '@wordpress/i18n';
+
+import { Component } from '@wordpress/element';
+
+import {
+	Button,
+	Dashicon,
+	Popover
+} from '@wordpress/components';
+
+/**
+ * External dependencies
+ */
+
+/**
+ * Internal dependencies
+ */
+
+import LocationSearch from '../location-search';
+
+import './style.scss';
+
+export class AddPoint extends Component {
+
+	constructor() {
+		super( ...arguments );
+		this.onAddPoint = this.onAddPoint.bind( this );
+	}
+
+	onAddPoint( point ) {
+
+		this.props.onAddPoint( point );
+
+	}
+
+	render() {
+		const {
+			onClose,
+			onAddPoint
+		} = this.props;
+		return (
+			<Button
+				className='map__add_btn'
```

Comment:
We usually use double quotes for string literals that we pass as props.

Reply:
Not sure I follow... where is a string literal being passed as props here?

Reply:
Oh, well `'map__add_btn'` is a literal of type string. Note that I mean string literal, not template literal (those 2 are often confused with each other).

Reply:
This was resolved (along with many other syntax issues) in https://github.com/Automattic/wp-calypso/commit/bca3b6c11f2bca13d7a57d781e8cc40f1c0810b1 after running all files through `JsPrettier`.